progress/progress.go

```diff
@@ -110,10 +101,13 @@ func (p *Progress) render() {
 	}
 
 	p.pos = len(p.states)
+	p.w.Flush()
 }
 
 func (p *Progress) start() {
 	p.ticker = time.NewTicker(100 * time.Millisecond)
+	// hide cursor and enable synchronized output
+	fmt.Fprint(p.w, "\033[?25l", "\033[?2026h")
```

Comment:
This way of using synchronized output is a bug.

Synchronized output mode is meant to be enabled per frame. What it does is to temporarily pause state updates. This code is requesting that the state never get updated through the entire duration of the progress bar. Fortunately, the terminal will just timeout the synchronized output mode and continue as normal, but that means that all this is accomplishing is delaying the first output until after that timeout period.

Reply:
Good to know. I'll revert it